app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [ ] [REQUIRED] While trying to signup, I encountered this error:

<img width="729" alt="Screenshot 2023-05-25 at 11 19 12" src="https://github.com/hienphan0111/Blog-app/assets/58692590/61962bcd-733d-44dd-b648-f3fe6d35ea2e">

Kindly provide the :host parameter, set default_url_options[:host], or set :only_path to true